debian/control

```diff
@@ -39,7 +39,9 @@ Build-Depends: bison,
                asciidoctor,
                libdw1,
                libdw-dev,
-               pkg-config
+               pkg-config,
+	       dwarves,
+	       xxd
```

Comment:
(nit) space formatting doesn't seem inline with previous entries.

My other question is, would that use the dwarves package that we added recently or the upstream ubuntu one? @prakashsurya

Reply:
The ubuntu one, IIRC.

Reply:
If we want to install ours, I think you need to modify linux-pkg to install it explicitly..

I don't believe we create an APT repository as part of linux-pkg, such that we can simply `apt-get install` linux-pkg package dependencies.. It'd probably be good if we did that, much like we do for appliance-build.. but don't think that's how it works today.

Reply:
the `dwarves` package isn't installed by default, so I think if we install here, the ubuntu version, it shouldn't cause any issues.. it's as if the delphix built dwarves packages doesn't exist..

Reply:
So do we want to address the `dwarves` dependency the `linux-pkg` config instead?

Reply:
Let's land this as-is.. if we want, we could later tweak linux-pkg to install the delphix package, and we wouldn't need to change this..